### ISSUE TITLE
Fit failing in 09-containerized-analysis (and earlier)

### ISSUE THREAD

**Issue by katilp** [awesome-workshop/intro-to-docker#6]
I have some furter problems for the fitting part, in CI/CD see https://gitlab.cern.ch/kati/awesome-analysis-statistics/-/jobs/7211434
```
File "fit.py", line 19, in main
37     data.Add(dataRunC)
38 AttributeError: 'TObject' object has no attribute 'Add'
```
The same happens in the container where all histogram files are available, after having done the skimming part under "Running the Containerized Workflow"

I copied over what is in https://gitlab.cern.ch/awesome-workshop/awesome-analysis-statistics-stage3/blob/master/.gitlab-ci.yml

**Comment by katilp** [comment]
Tried with rootproject/root-conda:6.18.04 in Dockerfile and .gitlab-ci.yml as well (as in the PR), but still [the same](https://gitlab.cern.ch/kati/awesome-analysis-statistics/-/jobs/7211981). I fail to see why this happens if it has run correctly in https://gitlab.cern.ch/awesome-workshop/awesome-analysis-statistics-stage3/pipelines
The only difference in the repos (as far as I can see) is that [my repo](https://gitlab.cern.ch/kati/awesome-analysis-statistics/tree/master) does not have .gitignore...

**Comment by katilp** [comment]
must be just a wrong input file - it happens also if I clone the original stage3 and the only change is the file. My file comes from an output of simple_fit from [an earlier skim-plot ci test job](https://gitlab.cern.ch/kati/awesome-analysis-eventselection/pipelines).

**Comment by danikam** [comment]
Hmm, strange. Unfortunately I don't have permission to download the file from your user area (`/eos/user/k/kati/awwstest/hist_ggH.root`), but one thing that's a bit striking is that the skim_ggH.root file I download from the artifacts of the [most recent skim-plot ci test job](https://gitlab.cern.ch/kati/awesome-analysis-eventselection/-/jobs/7206483) is 84 kB (and looks fine from what I can tell), whereas the [gitlab-ci pipeline shows the file being downloaded from your user area as 35 kB](https://gitlab.cern.ch/kati/awesome-analysis-statistics/-/jobs/7211434). I'm wondering if possibly the file upload to `/eos/user/k/kati/awwstest` just got interrupted, or the file was somehow otherwise corrupted?

**Comment by katilp** [comment]
OK, no worries, the fitting part in 09-containerized-analysis works when I finally have the histograms.root file properly available (see #7 )
